web_src/js/utils.test.js

```diff
@@ -132,3 +132,7 @@ test('toAbsoluteUrl', () => {
 
   expect(() => toAbsoluteUrl('path')).toThrowError('unsupported');
 });
+
+test('encodeURLEncodedBase64, decodeURLEncodedBase64', () => {
+  expect(encodeURLEncodedBase64(decodeURLEncodedBase64('foo'))).toEqual('foo');
```

Comment:
This test doesn't really test.

If you replace the `encodeURLEncodedBase64` with `encodeURIComponent` it also succeeds, it doesn't show the special part of `encodeURLEncodedBase64`

Reply:
There seems to be at least one difference over testing the same in the base functions:

```
> require("uint8-to-base64").encode(require("uint8-to-base64").decode("foo"))
'foo='

Reply:
@zeripath care to add a few test cases here? Don't have to be only round trip tests. You know best what data is expected.

Reply:
The round-trip `foo` was chosen specifically as it is different from what you would get with the from just using uint8-to-base64. I've added two other base64 urlencoded examples.

One problem with non-roundtrips is that the result of decodeURLEncodedBase64 is a Uint8Array and I am uncertain as to how I could test against that.